src/http-client.js

```diff
@@ -18,11 +23,31 @@ const makeQueryString = q =>
         .join('&')}`
     : ''
 
+/**
+ * Get API limits info from headers
+ */
+
+const responseHandler = res => {
```

Comment:
Can you replace this by

```js

const headersMapping = {
  'x-mbx-used-weight-1m': 'usedWeigh1m',
  ...
}

const responseHandler = res => {
  if (!res.headers || !res.url) {
    return
  }

  const marketName = res.url.includes(FUTURES) ? 'futures' : 'spot'

  Object.keys(headersMapping).forEach(key => {
    const outKey = headersMapping[key]

    if (res.headers.has(key)) {
      info[marketName][outKey] = res.headers.get(key)
    }
  })
}
```

Makes it easier to support more headers without repetition

Reply:
Done) Thanks for the nice solution tip)